vanilla-framework/vanilla-framework-react
# Combine button components into single component

## Done

Combine button components into single component

## QA

- Pull code
- Run `npm start`
- Check no errors
- View buttons at http://localhost:3000/

Fixes #1
